docs/doxygen/overviews/xrc_format.h

```diff
@@ -518,6 +518,28 @@ and can be one of the following "sub-properties":
 @endcode
 
 
+@subsection overview_xrcformat_type_extra_accels Accelerators List
+
+Defines a list of wxMenuItem's extra accelerators.
+
+The extra-accels property element is a "composite" element:
+it contains one or more <accel> "sub-properties":
```

Comment:
I think angle brackets need to be escaped (please run Doxygen locally to check):
```suggestion
it contains one or more @c \<accel\> "sub-properties":
```